src/constructs/rds/instance.test.ts

```diff
@@ -26,86 +26,135 @@ describe("The GuDatabaseInstance class", () => {
         version: PostgresEngineVersion.VER_11_8,
       }),
       app: "testing",
+      devXBackups: { enabled: true },
     });
 
     Template.fromStack(stack).hasResourceProperties("AWS::RDS::DBInstance", {
       DBInstanceClass: "db.t3.small",
     });
   });
 
-  it("sets the parameter group if passed in", () => {
+  test("sets the deletion protection value to true by default", () => {
     const stack = simpleGuStackForTesting();
-
-    const parameterGroup = new ParameterGroup(stack, "MyParameterGroup", {
-      parameters: { max_connections: "100" },
+    new GuDatabaseInstance(stack, "DatabaseInstance", {
+      vpc,
+      instanceType: "t3.small",
       engine: DatabaseInstanceEngine.postgres({
         version: PostgresEngineVersion.VER_11_8,
       }),
+      app: "testing",
+      devXBackups: { enabled: true },
     });
 
+    Template.fromStack(stack).hasResourceProperties("AWS::RDS::DBInstance", {
+      DeletionProtection: true,
+    });
+  });
+
+  test("sets DeleteAutomatedBackups to false by default", () => {
+    const stack = simpleGuStackForTesting();
     new GuDatabaseInstance(stack, "DatabaseInstance", {
       vpc,
       instanceType: "t3.small",
       engine: DatabaseInstanceEngine.postgres({
-        version: PostgresEngineVersion.VER_11_8,
+        version: PostgresEngineVersion.VER_16,
       }),
-      parameterGroup,
       app: "testing",
+      devXBackups: { enabled: true },
     });
 
     Template.fromStack(stack).hasResourceProperties("AWS::RDS::DBInstance", {
-      DBParameterGroupName: {
-        Ref: Match.stringLikeRegexp("MyParameterGroup[A-Z0-9]+"),
-      },
+      DeleteAutomatedBackups: false,
     });
   });
 
-  it("creates a new parameter group if parameters are passed in", () => {
+  test("adds the correct tag if the user opts-in to DevX Backups", () => {
     const stack = simpleGuStackForTesting();
     new GuDatabaseInstance(stack, "DatabaseInstance", {
       vpc,
       instanceType: "t3.small",
       engine: DatabaseInstanceEngine.postgres({
-        version: PostgresEngineVersion.VER_11_8,
+        version: PostgresEngineVersion.VER_16,
       }),
-      parameters: { max_connections: "100" },
       app: "testing",
+      devXBackups: { enabled: true },
     });
-
-    const template = Template.fromStack(stack);
-
-    template.hasResourceProperties("AWS::RDS::DBInstance", {
-      DBParameterGroupName: {
-        Ref: Match.stringLikeRegexp("DatabaseInstanceTestingParameterGroup[A-Z0-9]+"),
-      },
+    console.log(JSON.stringify(stack.tags.tagValues()));
+    GuTemplate.fromStack(stack).hasGuTaggedResource("AWS::RDS::DBInstance", {
```

Comment:
We should add functionality to check for a single tag, but no worries here! This would encourage good test practice (one assertion per test), and eliminate noise from cases like this one where we are focused on a single tag.

Reply:
@adamnfish - there's a suggestion for this here: https://github.com/guardian/cdk/pull/2280.